src/main/java/ch/puzzle/eft/controller/SiteController.java

```diff
@@ -48,4 +56,19 @@ public String viewValidatePage(@Valid ExamNumberForm examNumberForm, BindingResu
         }
         return SEARCH_TEMPLATE;
     }
-}
+
+
+    @PostMapping("/")
+    public ResponseEntity<String> acceptCookies(Model model) {
+        ResponseCookie.ResponseCookieBuilder cookieBuilder = ResponseCookie.from("cookie-consent", "true");
+        cookieBuilder.maxAge(60 * 60 * 24 * 365);
+        cookieBuilder.sameSite("Strict");
+        cookieBuilder.httpOnly(true);
```

Comment:
may be deleted or set to false, to be checked.

Reply:
Removed it because it did not change anything